test/claw_reference_test.cmake

```diff
@@ -456,10 +441,7 @@ function(claw_add_advanced_test)
       WORKING_DIRECTORY ${claw_add_advanced_test_WORKING_DIRECTORY}
       DEPENDS ${MAIN_F90} ${OUTPUT_FILE_ACC} ${OUTPUT_FILE_EXTRA_ACC}
       COMMENT "${CLAWFC} for GPU target with ACC directive on file ${MAIN_F90}"
-    )
 
-    # Execute the CLAW Compiler for GPU target with OMP directive
-    add_custom_command(
```

Comment:
@peclatj It's not what I meant in my previous comment. Just adding a dependency should have solved the problem.